utils/supportedLanguagesUtils.test.js

```diff
@@ -0,0 +1,134 @@
+const {
+  supportedLanguages,
+  getLanguagesCodesWithNames, 
+  validateLanguageIsSupportedByEngine,
+  validateLanguageRequested,
+  getLanguageCodeByEngine,
+} = require("./supportedLanguagesUtils");
+
+describe("getLanguagesCodesWithNames", () => {
```

Comment:
Also moved all the tests of the functions to this file! I separated the describes by functions so is easy read the test results